torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -128,15 +128,41 @@ def _retrieve_or_adapt_input_to_graph_set(
                 List[int],
             ]
         ] = []
+        # onnx_tensor contains a list of scalars which could be one of
```

Comment:
In line 108, it could be sym type in node.meta['val'], and most likely calculated by `BuiltinFunctionType` (See onnxfunction_dispatcher.py line 321). I am not sure if that's also in the scope in this PR. The changes you made seems only for Sequence.

Reply:
https://github.com/pytorch/pytorch/blob/c30346db0e89a58b6ab7121048e7557af235b584/torch/onnx/_internal/fx/onnxfunction_dispatcher.py#L321-L339

Reply:
This is for sequence-only. The change I made is in this if-block:
```py
    elif isinstance(onnx_tensor, (tuple, list)) and any(
        isinstance(node, torch.fx.Node)
        and fx_type_utils.is_torch_symbolic_type(node.meta.get("val"))
        for node in onnx_tensor
    ):
```